release_notes.md

```diff
@@ -1,55 +1,84 @@
 # What's New
 
-## Enhancement
-
-- FortiSOAR Minimum Compatibility - 7.4.1
-- Renamed playbook `Set Assets With MEF Tag` to `Tag as Most Essential Function (MEF)` and added manual input to get reason for tagging asset as Most Essential Function (MEF).
-- Introduced new **Asset Icon** record, `VOIP Phone`, `Gateway`, `Controller`, `Historian`, and `Printer`
-- Introduced new playbook `Set Asset Icon` under collection **02 - Use Case - IT OT Asset Management** which correlate **Asset Icon** with **Asset** as per asset category.
-- Dashboard:
-  - Configured Record Distribution widget under `Asset Overview` with title as Asset Hostname.
-  - Introduced new widget **Top 10 Devices With Vulnerabilities** under **Asset Risk Overview**
+> The minimum compatibility of this solution pack with FortiSOAR&trade; has been enhanced to FortiSOAR v7.4.1 and later.
+
+## Playbook Enhancements
+
+- A new playbook **Set Asset Icon** under the collection **02 - Use Case - IT OT Asset Management** correlates *Asset Icon* with *Asset* as per the asset's category.
+
+- The playbook *Set Assets With MEF Tag* has been renamed to **Tag as Most Essential Function (MEF)**.
+    - A manual input now asks for a reason when tagging assets as MEF.
+
+- Renamed the playbook *Links MITRE Matrices To Alert* to **Links Assets To Alert**
+    - Removed the correlation with MITRE module data as the correlation is now a part of **MITRE ATT&CK Enrichment Framework** v2.2.0 and later.
+
+## Record Enhancements
+
+- A new record **Asset Icon** now has icons for following asset categories:
+    - VOIP Phone
+    - Gateway
+    - Controller
+    - Historian
+    - Printer
+
+## Dashboard Enhancements
+
+- The **Asset Risk Overview** dashboard now appears with following enhancements:
+
+    - A section lists **Top 10 Devices With Vulnerabilities**
+
+    - A section **Recent Alerts On Critical Assets** comes with pre-defined filter to list assets based on their criticalities &ndash; *Critical* or *Very Critical*
+
```

Comment:
Can we call 'widgets' instead of 'section' ?

Reply:
**Recent Alerts on Critical Assets** is a card tile widget. Calling it something else may trigger users to search for that widget on Content Hub